app/lib/actions/candlepin/consumer/remove_subscription.rb

```diff
@@ -0,0 +1,18 @@
+module Actions
+  module Candlepin
+    module Consumer
+      class RemoveSubscription < Candlepin::Abstract
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        input_format do
+          param :uuid, String
+          param :entitlement_id, String
```

Comment:
yes, entitlement_id is different.  'Entitlement' is the object in candlepin that joins the pool with a consumer, having a quantity